README.md

```diff
@@ -1,4 +1,77 @@
-# hubrise-webclient
-A pet JS/HTML project to see new HubRise orders in a web page
+<a href="https://www.hubrise.com/"><img src="./screenshots/hubrise-logo.png" align="left" height="48"></a>
+<br><br><br>
 
-[Demo](http://philatist-fish-74534.netlify.com)
+# HubRise {{app_name}} Template
```

Comment:
"EasyTablet", "TinyTablet", "HelloWorld", "TinyOrder", "MiniOrder"... Any suggestions?

Reply:
I'd go for TinyTablet!